### DISCUSSION TITLE
Why is pagination not working

### DISCUSSION THREAD

**Discussion by hsiraaj** [rappasoft/laravel-livewire-tables#471] [Q&A] [answered]
I have a table with more than 10 records which by default per page it show 10 records. If i select 20 or All from dropdown option it shows all the records, however the pagination which should show the current page total and next, previous options  is not showing. 

all options are enabled:
 public bool $perPageAll = true;
    public bool $paginationEnabled = true;
    public bool $showPerPage = true;
    public bool $showPagination = true;

<img width="1406" alt="Screenshot 2021-09-28 at 16 58 23" src="https://user-images.githubusercontent.com/16245787/135082900-4128e66c-f762-4213-913f-6118388b199f.png">

below here is what I want: but couldn't make it to work or show it
<img width="904" alt="Screenshot 2021-09-28 at 17 02 44" src="https://user-images.githubusercontent.com/16245787/135083288-756531f1-8bfe-4dd5-a458-95d4689e2e94.png">

**Comment by rappasoft** [comment, marked as answer]
It's enabled by default. The only of those options you need is `$perPageAll` since that is off by default. If you're not seeing it then something else is wrong.

**Reply by hsiraaj** [reply]
found the issue.. in my case there was a hidden class "md:hidden" which I have no idea how it got there.. I just removed it and everything looks good!

Thank you for the awesome package!
